Client/Frontend/Browser/Playlist/Controllers/PlaylistMoveFolderView.swift

```diff
@@ -0,0 +1,269 @@
+    ) var folders: FetchedResults<PlaylistFolder>
+    
+    @FetchRequest(
+        entity: PlaylistFolder.entity(),
+        sortDescriptors: [],
+        predicate: NSPredicate(format: "uuid == %@", PlaylistFolder.savedFolderUUID)
+    ) var savedFolder: FetchedResults<PlaylistFolder>
+    
+    @State private var moveDisabled: Bool = true
+    @State private var selectedFolder = PlaylistManager.shared.currentFolder
+    
+    var selectedItems: [PlaylistItem]
+    var onCancelButtonPressed: (() -> Void)?
+    var onDoneButtonPressed: (([PlaylistItem], PlaylistFolder?) -> Void)?
+    
+    var body: some View {
+        NavigationView {
+            Form {
+                Section(header: {
+                    HStack(spacing: 18.0) {
+                        if selectedItems.count > 1,
+                           let firstItem = selectedItems[safe: 0],
+                           let secondItem = selectedItems[safe: 1] {
+                            ZStack(alignment: .topLeading) {
+                                PlaylistFolderImage(item: firstItem)
+                                    .rotationEffect(.degrees(5.0))
+                                PlaylistFolderImage(item: secondItem).rotationEffect(.degrees(-5.0))
+                            }
+                            
+                            Text("\(firstItem.name ?? "") & \(selectedItems.count - 1) item(s)")
+                                .lineLimit(2)
+                                .font(.body)
+                                .foregroundColor(.white)
+                                .textCase(.none)
+                                .fixedSize(horizontal: false, vertical: true)
+                        } else if let item = selectedItems.first {
+                            PlaylistFolderImage(item: item)
+                            Text(item.name ?? "")
+                                .lineLimit(2)
+                                .font(.body)
+                                .foregroundColor(.white)
+                                .textCase(.none)
+                        }
+                    }
+                    .padding(.top, 20.0)
+                }()) {}
+                .listRowInsets(.zero)
+                .listRowBackground(Color.clear)
+                
+                Section(header: Text(Strings.PlaylistFolders.playlistFolderMoveFolderCurrentSectionTitle)
+                            .font(.footnote)
+                            .textCase(.none)
+                            .foregroundColor(Color(.secondaryBraveLabel))
+                            .multilineTextAlignment(.leading)) {
+                    
+                    Button(action: {
+                        selectedFolder = PlaylistManager.shared.currentFolder
+                        moveDisabled = true
+                    }) {
+                        PlaylistFolderView(iconColor: .gray,
+                                           titleColor: .gray,
```

Comment:
As mentioned above, don't pass in colors, design is 50% opacity on title/subtitle and `braveDisabled` color for icon not gray

Reply:
Ah sorry I chose the light mode `braveDisabled` in Figma.

The colours are wrong though now, the the subtitle should be `secondaryBraveLabel`.  The opacity is applied based on if its the source folder can be applied to the whole `HStack`

Reply:
Fixed.